ios/KlippaScannerSDK.swift

```diff
@@ -0,0 +1,440 @@
+        }
+
+        if let imageMovingSensitivity = config["ImageMovingSensitivityiOS"] as? CGFloat {
+            builder.klippaImageAttributes.imageMovingSensitivity = imageMovingSensitivity
+        }
+
+        if let cropPadding = config["CropPadding"] as? [String: CGFloat] {
+            if let width = cropPadding["width"], let height = cropPadding["height"] {
+                builder.klippaImageAttributes.cropPadding = CGSize(width: width, height: height)
+            }
+        }
+
+        if let storeImagesToCameraRoll = config["StoreImagesToCameraRoll"] as? Bool {
+            builder.klippaImageAttributes.storeImagesToCameraRoll = storeImagesToCameraRoll
+        }
+
+        if let shutterButton = config["ShutterButton"] as? [String: Bool] {
+            if let allowShutterButton = shutterButton["allowShutterButton"] {
+                builder.klippaShutterbutton.allowShutterButton = allowShutterButton
+            }
+
+            if let hideShutterButton = shutterButton["hideShutterButton"] {
+                builder.klippaShutterbutton.hideShutterButton = hideShutterButton
+            }
+        }
+
+        if let previewDuration = config["PreviewDuration"] as? Double {
+            builder.klippaDurations.previewDuration = previewDuration
+        }
+
+        if let successMessage = config["Success"] as? [String: Any] {
+            if let successMessage = successMessage["message"] as? String {
+                builder.klippaMessages.successMessage = successMessage
+            }
+
+            if let previewDuration = successMessage["previewDuration"] as? Double {
+                builder.klippaDurations.successPreviewDuration = previewDuration
+                builder.klippaDurations.previewDuration = 0
+            }
+        }
+
+        setBuilderObjectDetectionModel(config, builder)
+        setBuilderCameraModes(config, builder)
+        
+        return builder
+    }
+
+    fileprivate func setBuilderObjectDetectionModel(_ config: [String: Any], _ builder: KlippaScannerBuilder) {
+
+        guard let model = config["Model"] as? [String: String] else {
+            return
+        }
+
+        guard let modelFile = model["modelFile"] else {
+            return
+        }
+
+        guard let modelLabels = model["modelLabels"] else {
+            return
+        }
+
+        builder.klippaObjectDetectionModel = KlippaObjectDetectionModel(
+            modelFile: modelFile,
+            modelLabels: modelLabels
+        )
+
+    }
+
+    fileprivate func setBuilderCameraModes(_ config: [String: Any], _ builder: KlippaScannerBuilder) {
+        var modes = [KlippaDocumentMode]()
+
+        if let cameraModeSingle = config["CameraModeSingle"] as? [String: Any] {
+
+            let single = KlippaSingleDocumentMode()
+
+            if let name = cameraModeSingle["name"] as? String {
+                single.name = name
+            }
+
+            let instructions = Instructions(message: "Single Document",
```

Comment:
You're hardcoding the instructions message instead of using the `message` inside of the interface provided. This applies to each document mode instructions.